broadinstitute/ddp-study-server
# Fixing proxy columns and grouped option exports

This fixes the export of grouped options, and proxy columns.  

TO TEST:
1. open DSM to pancan study participant list
2. customize view, and select the "Your/Your child's DIAGNOSIS TYPE" -> "DIAGNOSIS TYPE" column
3. Download the export with default options selected
4. confirm that the DIAGNOSIS_TYPE column in the export file shows human-readable text, such as "Colon cancer" or "Retinoblastoma (RB)"

5. Select the columns "Participant Columns" -> email  and "Proxy columns" -> email
6. download export with default options selected
7. confirm that profile.email column and profile.PROXY.email columns are populated correctly.
